tgui/packages/tgui/interfaces/PrisonerShuttleConsole.js

```diff
@@ -0,0 +1,64 @@
+import { useBackend } from '../backend';
+import {
+  Button,
+  ProgressBar,
+  LabeledList,
+  Section,
+} from '../components';
+import { Window } from '../layouts';
+
+export const PrisonerShuttleConsole = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    can_go_home,
+    emagged,
+    id_inserted,
+    id_name,
+    id_points,
+    id_goal,
+  } = data;
+  const bad_progress = emagged ? 0 : 1;
+  const completionStatus = emagged
+    ? 'ERR0R'
+    : can_go_home
+    ? 'Completed!'
+    : 'Insufficient';
```

Comment:
```suggestion
  const completionStatus = can_go_home : 'Completed!' ? 'Insufficient';
  if (emagged) {
    completionStatus = 'ERR0R'
  }
```
logic nitpick here, the second ternary here should have been indented to indicate it was nested, instead of doing that I want to move this over to an if statement. It will be a lot more readable.

Reply:
You can't do this how you suggest, as `completionstatus` is a contant, editing it after declaration will cause a runtime.

https://eslint.org/docs/latest/rules/no-const-assign

Reply:
op you're right, just use `let` in this context

Reply:
Applied a modified version of this @Sirryan2002 since i think you got : and ? mixed up

```
let completionStatus = can_go_home ? 'Completed!' : 'Insufficient';
  if (emagged) {
    completionStatus = 'ERR0R'
  }
```